src/Frontend/state/actions/resource-actions/__tests__/preference-actions.test.ts

```diff
@@ -155,4 +156,53 @@ describe('getOriginIdsToPreferOver', () => {
     const expectedOriginUuids = ['originUuid1'];
     expect(actualOriginUuids).toEqual(expectedOriginUuids);
   });
+
+  it('finds all origin ids for attribution with multiple root resources', () => {
+    const testSource = faker.opossum.source();
+    const file1 = faker.opossum.resourceName();
+    const file2 = faker.opossum.resourceName();
+    const pathFile1 = `/${file1}`;
+    const pathFile2 = `/${file2}`;
+    const uuid1 = faker.string.uuid();
+    const uuid2 = faker.string.uuid();
+    const resources: Resources = faker.opossum.resources({
```

Comment:
the function is already fully typed so there is no need for re-declaring the type
```suggestion
    const resources = faker.opossum.resources({
```